embark-consult.el

```diff
@@ -181,6 +181,37 @@ This function is meant to be added to `embark-collect-mode-hook'."
       (when (fboundp 'wgrep-setup) (wgrep-setup)))
     (pop-to-buffer buf)))
 
+(defun embark-consult-export-lines-to-grep (lines)
```

Comment:
Any ideas for a better name?

Reply:
I'll update the name after refactoring. This was my "code name" because I was looking for a solution specifically for lines. I didn't draw much difference from `consult-location` group back when I started my initial solution discovery.